src/apps/Altinn.Authorization/src/Altinn.Authorization/Controllers/DecisionController.cs

```diff
@@ -380,6 +421,23 @@ private async Task<bool> IsAccessListAuthorized(XacmlContextRequest decisionRequ
             return true;
         }
 
+        private static XacmlContextRequest GetRequestForLog(string input, bool isJson)
+        {
+            XacmlContextRequest result;
+            if (isJson)
+            {
+                XacmlJsonRequestRoot jsonRequest = JsonSerializer.Deserialize<XacmlJsonRequestRoot>(input, jsonSerializerOptions);
+                result = XacmlJsonXmlConverter.ConvertRequest(jsonRequest.Request);
+            }
+            else
+            {
+                using XmlReader reader = XmlReader.Create(new StringReader(input));
```

Comment:
Ref CodeQL warning så er jeg usikker på om vi skal bry oss med logging av XML requests her. Håper strengt tatt ingen bruker det lenger (som for så vidt hadde vært nice og hatt noe statistikk/log av 🤔 )